make/clean.mk

```diff
@@ -1,4 +1,12 @@
 .PHONY: clean
 clean:
 	$(Q)-rm -rf ${V_FLAG} $(OUT_DIR) ./vendor
-	$(Q)go clean ${X_FLAG} ./...
\ No newline at end of file
+	$(Q)go clean ${X_FLAG} ./...
+
+.PHONY: clean-resources
+clean-resources:
```

Comment:
It's a bit weird target... It deletes usersignups, all pods (including the operator pods, but not deployments) and cluster resource quotas only. What's the purpose of such target?
If you want to introduce a target to delete all resources created by usersignups (usersignup and all child resources) then shouldn't deleting the usersignups only be enough? Cluster resource qouatas are "child" objects too and should be deleted by GC. And why pods?

Reply:
this is a target I added while doing trying things on a temp cluster. Deleting the pods meant I would start with fresh pods and hence, so it's a trick to clear the logs, actually ;)
but I can remove this target as it was really tailored for my own testing.

Reply:
But why not using `make clean-e2e-resources` for that?

Reply:
> But why not using make clean-e2e-resources for that?

because `clean-e2e-resources` deletes the whole deployment of the operator including the `CatalogSource`. This target is just to delete the internal toolchain objects (via deleting `usersignups` and deleting `clusterresourcequotas` specifically) plus retriggering the deployment of the operators (via deleting the pod). In addition, this can be used only with the dev namespaces (namespace name without the time suffix) otherwise, it wouldn't work. 

I understand the purpose of the target, but I would do some additional changes there:
* rename for example to `clean-toolchain-resources-in-dev`
* change `HOST_NS` to `DEV_HOST_NS` and `MEMBER_NS` to `DEV_MEMBER_NS`
* when the usersignups are deleted then I would delete the rest of the toolchain objects (including BannedUser, NSTemplateTier, etc.)
* for `xargs oc delete` skip the empty values so it doesn't fail when there is no resource found.

I'll send you a suggestion in the next comment ;-)